commit 4b65339e472290a5b0cd4c7352ecc80e23f776df
Author: Trent Mick <trentm@gmail.com>
Date:   2019-07-19T10:16:39-07:00 (3 months ago)
    
    TRITON-1824 make mockcloud and sdc-system-tests builds upload to public builds area
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
    Approved by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/Makefile b/Makefile
index 9004c06..90b533e 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@
 #
 
 #
-# Copyright (c) 2019, Joyent, Inc.
+# Copyright 2019 Joyent, Inc.
 #
 
 #
@@ -31,9 +31,6 @@ else
 	NPM	:= $(shell which npm)
 endif
 
-# sdc-system-tests is not a public module, so override
-ENGBLD_DEST_OUT_PATH ?= /stor/builds
-
 ENGBLD_REQUIRE := $(shell git submodule update --init deps/eng)
 include ./deps/eng/tools/mk/Makefile.defs
 TOP ?= $(error Unable to access eng.git submodule Makefiles.)
